Sources/Momento/CacheClient.swift

```diff
@@ -1,11 +1,15 @@
 import Foundation
 
-public enum ScalarType {
+enum ScalarType {
     case string(String)
     case data(Data)
 }
 
-typealias CacheClientProtocol = ControlClientProtocol & DataClientProtocol
+// The CacheClient class provides user-friendly, public-facing methods
+// with default values for non-required parameters and overloaded functions
+// that accept String and Data values directly rather than ScalarTypes
+
+protocol CacheClientProtocol: ControlClientProtocol & DataClientProtocol {}
```

Comment:
CacheClientProtocol is back in, but the protocol extension wasn't needed and wasn't working as expected.

The overloaded methods for accepting String and Data directly are implemented as part of the CacheClient class, and the methods from DataClientProtocol (using ScalarType) are implemented as internal functions that the overloaded ones call